addons/survey/wizard/survey_invite_views.xml

```diff
@@ -5,27 +5,30 @@
             <field name="name">survey.invite.view.form</field>
             <field name="model">survey.invite</field>
             <field name="arch" type="xml">
-                <form string="Compose Email" class="o_mail_composer_form">
+                <form string="Compose Email" class="o_mail_composer_form" style="padding-bottom: 0px;">
                     <group col="1">
                         <group col="2">
                             <field name="survey_access_mode" invisible="1"/>
                             <field name="survey_users_login_required" invisible="1"/>
                             <field name="survey_users_can_signup" invisible="1"/>
-                            <field name="survey_id" readonly="context.get('default_survey_id')"/>
+                            <field name="survey_id" readonly="context.get('default_survey_id')" invisible="1"/>
```

Comment:
Do we still need the readonly tag if it's invisible now?

Reply:
It is indeed needed for the creation of the invite, if it is not present we get a traceback.
But this traceback may be linked to the `_render_field` call so if I remove the call and either make the template subject static or if we go back to the old method, this field will be removed.

Reply:
(Just to clarify, I was suggesting removing the readonly attribute, not the entire field)

Reply:
I also think the readonly is not necessary anymore :)